config/cms.php

```diff
@@ -93,6 +93,19 @@
 
     'backendSkin' => 'Backend\Skins\Standard',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Determines if logging in backend should run UpdateManager
+    |--------------------------------------------------------------------------
+    |
+    | If value is true it runs UpdateMananger after login in backend.
```

Comment:
@Samuell1 Grammar fix.
```suggestion
    | If value is true, UpdateMananger will be run on logging in to the backend.
```